### PR TITLE
Transparency/Alpha show on Material Scans

### PR DESCRIPTION
**Pull request by 444Portal**
## About the PR
shows alpha on material scans with extremely, very, transparent and opaque as the describers. same criteria used for the capgun is used to judge the alpha

## Why's this needed? 
now that alpha is used for something more than style i think we can put it in the scanners, will help builders and captains wanting to shoot twice the gun

## Changelog

```changelog
(u)444explorer
(+)Material Scanners now show the Materials Alpha/Transparency value
```

### CODE REVIEW

File: code/obj/item/device/pda2/scanners.dm [hunk 1]
```diff
@@ -184,6 +184,15 @@
 				for(var/datum/material_property/mat in A.material.getMaterialProperties())
 					var/value = A.material.getProperty(mat.id)
 					. += "• [mat.getAdjective(A.material)] ([value])<br>"
+				switch(A.material.getAlpha())
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/scanners.dm:187]: Why aren't you processing like the rest with get adjective?

Comment (444Portal) [reply code/obj/item/device/pda2/scanners.dm:187]: Err, good point now that you mention it - from my limited knowledge though that'd need a datum/property that stores alpha that i can stick getAdjective on - should I do that?

Comment (TobleroneSwordfish) [reply code/obj/item/device/pda2/scanners.dm:187]: Ideally yes, you could preserve `getAlpha` as a wrapper proc to access the new opacity property's value though.

File: code/modules/materials/Mat_Processing.dm [hunk 2]
```diff
@@ -266,6 +266,15 @@ TYPEINFO(/obj/item/device/matanalyzer)
...
 						boutput(user, SPAN_NOTICE("• [X.getAdjective(W.material)] ([value])"))
+					switch(W.material.getAlpha())
+						if(-INFINITY to 80)
+							boutput(user, SPAN_NOTICE("• extremely transparent ([W.material.getAlpha()])"))
+						if(80 to 130)
+							boutput(user, SPAN_NOTICE("• very transparent ([W.material.getAlpha()])"))
+						if(130 to 180)
+							boutput(user, SPAN_NOTICE("• transparent ([W.material.getAlpha()])"))
+						if(180 to INFINITY)
+							boutput(user, SPAN_NOTICE("• opaque ([W.material.getAlpha()])"))
```
Comment (TobleroneSwordfish) [inline code/modules/materials/Mat_Processing.dm:277]: Might read better if it went "opaque, slightly transparent, transparent, very transparent"

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request closed without merging**